Cython/Utility/Overflow.c

```diff
@@ -46,6 +46,22 @@ static int __Pyx_check_twos_complement(void) {
 #define __Pyx_div_no_overflow(a, b, overflow) ((a) / (b))
 #define __Pyx_div_const_no_overflow(a, b, overflow) ((a) / (b))
 
+#if defined(__has_builtin)
+#  if __has_builtin(__builtin_add_overflow) && !defined(__ibmxl__)
+#    define HAVE_BUILTIN_OVERFLOW
```

Comment:
I think most macros like this start with `CYTHON_` (e.g. `CYTHON_HAVE_BUILTIN_OVERFLOW`), just to reduce the chances of conflicting with someone else's macro.